config/default.yml

```diff
@@ -59,3 +50,15 @@ Ezcater/StyleDig:
 Ezcater/RubyTimeout:
   Description: "Disallow use of `Timeout.timeout` because it is unsafe and can cause unexpected behavior."
   Enabled: true
+
+GraphQL/FieldDescription:
+  Enabled: false
+GraphQL/ArgumentDescription:
+  Enabled: false
```

Comment:
Tangential: I would love for us to adopt these at some point.
It would be great if we could also adopt the no-self-references rule from the menus team for these as well when/if we adopt these in the future.

Reply:
No day like today 🤷 Given that this is already a breaking change, this may be the best time to roll it out if you're so inclined.